packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1195,9 +1195,9 @@ export function createLodestarMetrics(
         help: "Histogram of cloned count per state every time state.clone() is called",
         buckets: [1, 2, 5, 10, 50, 250],
       }),
-      statePersistDuration: register.histogram({
-        name: "lodestar_cp_state_cache_state_persist_seconds",
-        help: "Histogram of time to persist state to db",
+      stateSerializeDuration: register.histogram({
+        name: "lodestar_cp_state_cache_state_serialize_seconds",
```

Comment:
There are few other metrics we could consider renaming, those are still unused (not part of dashboard)

https://github.com/ChainSafe/lodestar/blob/ba57d4b655bd229b7271504b518f6ad3ba9a0b23/packages/beacon-node/src/metrics/metrics/lodestar.ts#L1212

But rather part of another PR

Reply:
yes will persist all metrics in a follow-up PR